acra/acra-in-depth/architecture/key-storage-and-kms/key-storage.md

```diff
@@ -51,4 +51,4 @@ We recommend deploying a dedicated Redis database as key storage and token stora
 
 When AcraServer/AcraTranslator/AnyProxy need a key, they ask key storage for it. Each key has its purpose (transport, encryption, audit log, HMAC) and is bound to a specific client ID or zone. Read more about [key management](/acra/security-controls/key-management/).
 
-AcraServer and AcraTranslator will require key storage for tokenization. When [tokenization](/acra/security-controls/tokenization/) is performed, the key storage will contain data needed for consistent tokenization (return same result for same input next time) and for detokenization (reverse operation).
+AcraServer and AcraTranslator will require key storage for tokenization. When [tokenization](/acra/security-controls/tokenization/) is performed, the key storage will contain data needed for consistent tokenization (return same result for same input next time) and for de-tokenization (reverse operation).
```

Comment:
we use `detokenization` term everywhere, both are ok, `detokenization` is more popular and it's across all the docs and website

<img width="825" alt="Screenshot 2021-11-29 at 21 33 45" src="https://user-images.githubusercontent.com/2877920/143930949-3b99b970-d9bd-417f-9778-a70dcc6febcb.png">

Reply:
Agree with such argument